### ISSUE TITLE
1.2 and 1.3 beta doesnt connect over ssh with rsa algorithm

### ISSUE THREAD

**Issue by vladkens** [Studio3T/robomongo#1623]
Hi. I was try to connect to my database on remote server through ssh using RSA algorithm and always have error:
```
Failed to create SSH tunnel to HOST.

Error:
Resource temporarily unavailable. Authentication by key (LINK TO KEY) failed (Error -16). (Error #35)
```

I generate key like this
```ssh-keygen -o -t rsa -b 4096 -C "email@example.com"```

Also I was try use 2048 and 1024 bits version but have same result – connect failed.

I know that this is common problem with Robo3T and Studio.

Which kind of information you need more to fix this issues and make application better?

**Comment by sagivle** [comment]
same issue 4096 not connecting, 2048 working fine

**Comment by simsekgokhan** [comment]
Hi @vladkens , @sagivle , thanks for reporting the problem and sorry for answering so late.  

On Windows & macOS, we have upgraded our SSH library to the latest version (libssh2 v1.9.0 - Jun/2019) and it supports ECDSA and Ed25519 keys. Also during development, we saw that the upgrade fixed some problematic cases. 
I hope Robo 3T 1.4 will fix some of your problems -> [Robo 3T 1.4](https://github.com/Studio3T/robomongo/releases/tag/v1.4.0)

Please also see: https://github.com/Studio3T/robomongo/issues/1189#issuecomment-447109457

**Comment by sagivle** [comment]
Thanks
